package.json

```diff
@@ -23,7 +23,7 @@
     "postcss": "8.4.26",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "tailwind-merge": "^1.14.0",
```

Comment:
Just a quick heads-up: the `tailwind-merge` dependency is actually important for the `cn utility function`. Removing it might lead to bugs and merge conflicts with the development branch. It would be better to keep it in the PR. Thanks! 🚀